runtime/altair/src/lib.rs

```diff
@@ -1940,6 +1940,7 @@ pub type SignedExtra = (
 	frame_system::CheckWeight<Runtime>,
 	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
 	runtime_common::transfer_filter::PreBalanceTransferExtension<Runtime>,
+	frame_metadata_hash_extension::CheckMetadataHash<Runtime>,
```

Comment:
NIT: It should not make a difference as `runtime_common::transfer_filter::PreBalanceTransferExtension<Runtime>::Extra` is `()`, but could we switch the two? Then we have the same order as the polkadot chains IIRC. 

Same for all other runtimes

Reply:
[22d0a38](https://github.com/centrifuge/centrifuge-chain/pull/1865/commits/22d0a3898a9622424dac70799120839847bfec41)